### PR TITLE
Issue 15 - Lazy load twitter element

### LINKED ISSUE #15: Lazy load tweets
We need to lazy load tweets embedded.  In our use case there are hundreds of tweets into the same page, that is loaded at once.

We would like to have the lazy load functionality working with these tweets, and load the tweets when needed.

So to make it happen we need to:
1) Transform the html to insert widgets.js just one time.
2) Change all tweet class to another class (or keep into data-class attribute)
3) When load one tweet, a new lazyload plugin should fix the tweet class just for the displayed tweet and fire the ``twttr.widgets.load()`` method to load the tweet.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #15

### CODE REVIEW

**Comment by rodfersou** [comment]
@hvelarde the transform is not working with liveblog.. the blockquote elements are not processed for some reason.. can you help me?

**Comment by rodfersou** [comment]
@hvelarde it looks like it is loading all tweets anyway.. the class change didn't solve the problem

**Comment by rodfersou** [comment]
no.. I got another problem here, and the transform is not working again..

**Comment by rodfersou** [comment]
Nevermind, I just discovered that the transform don't work when access from localhost (is it expected?).
When I access from my dynamic dns site it worked like a charm!

**Comment by rodfersou** [comment]
@hvelarde now I remove the widgets.js (just realized that plone was configured to allow script tags into tinymce in production site)

**Comment by hvelarde** [comment]
@rodfersou I made some changes to the code (I still need to increase test coverage):

* the plugin must be loaded before the main script
* the transformer was enhanced and split in many functions to make it easier to test the use cases

@aFarkas sorry for the delay giving you feedback on this but we have been quite busy.

File: src/collective/lazysizes/upgrades/v4/__init__.py [hunk 1]
```diff
@@ -1 +1,23 @@
...
+    js_tool.registerScript(
+        twitter_script,
+        cacheable=True,
+        compression='none',
+        cookable=True,
+        enabled=True,
+        expression='',
+        inline=False,
+    )
+    js_tool.moveResourceBefore(twitter_script, main_script)
```
Comment (rodfersou) [inline src/collective/lazysizes/upgrades/v4/__init__.py]: This could be handled with generic setup ``insert-after`` attribute

**Comment by hvelarde** [comment]
@aFarkas I'll close the issue on your repo when the code is in production to leave a link so people can see it working; thanks for your help!

**Pull request merged**